course/src/isometry.cc

```diff
@@ -0,0 +1,82 @@
+#include "isometry.h"
+
+
+namespace cppcourse {
+
+const Vector3 Vector3::kUnitX = {1., 0., 0.};
+const Vector3 Vector3::kUnitY = {0., 1., 0.};
+const Vector3 Vector3::kUnitZ = {0., 0., 1.};
+const Vector3 Vector3::kZero = {0., 0., 0.};
+
+
+Vector3 Vector3::operator+(const Vector3 &other) const{
+    return Vector3(x_ + other.x_, y_ + other.y_, z_ + other.z_);
+};
+Vector3 Vector3::cross(const Vector3 & other) const{
+    Vector3 output = Vector3::kZero;
+    output.x_ = y() * other.z() - z() * other.y();
+    output.y_ = z() * other.x() - x() * other.z();
+    output.z_ = x() * other.y() - y() * other.x();
+    return output;
+}
+Vector3 Vector3::operator/(const Vector3 &other) const{
+    return Vector3(x_ / other.x_, y_ / other.y_, z_ / other.z_);
+}
+Vector3 Vector3::operator-(const Vector3 &other) const{
+    return Vector3(x_ - other.x_, y_ - other.y_, z_ - other.z_);
+};
+Vector3 Vector3::operator*(const Vector3 &other) const{
+    return Vector3(x_ * other.x_, y_ * other.y_, z_ * other.z_);
+};
+Vector3 Vector3::operator*(const double &other) const{
+    return Vector3(x_ * other, y_ * other, z_ * other);
+};
+bool Vector3::operator==(const Vector3 &other) const{
+    return (x_ == other.x_ && y_ == other.y_ && z_ == other.z_);
+}
+Vector3::Vector3(const std::initializer_list<double> &rhs){
+    auto begin_ptr = rhs.begin();
+    int i = 0;
```

Comment:
As discussed yesterday, avoid using this index variable.

Reply:
Addressed